SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -63,6 +64,8 @@ public sealed class SlowCheetahPackage : Package
         public static SlowCheetahPackage OurPackage { get; set; }
         private string pkgName = Settings.Default.SlowCheetahNugetPkgName;
 
+        private IList<string> TempFilesCreated { get; set; }
```

Comment:
Instead of setting this in `Initialize` you can change it to `private IList<string> TempFilesCreated { get; } = new List<string>();`

Then you also don't need to null-check it in dispose. You actually don't need to null check it already.